cmd/release-controller-api/http.go

```diff
@@ -153,12 +163,272 @@ func (c *Controller) userInterfaceHandler() http.Handler {
 	mux.HandleFunc("/api/v1/releasestreams/rejected", c.apiRejectedStreams)
 	mux.HandleFunc("/api/v1/releasestreams/all", c.apiAllStreams)
 
+	mux.HandleFunc("/api/v1/features/{release}/release/{tag}", c.apiFeatureReleaseInfo)
+	mux.HandleFunc("/features/{release}/release/{tag}", c.httpFeatureReleaseInfo)
+
 	// static files
 	mux.PathPrefix("/static/").Handler(http.StripPrefix("/static/", http.FileServer(http.FS(resources))))
 
 	return mux
 }
 
+func (c *Controller) featureReleaseInfo(tagInfo *releaseTagInfo) ([]*FeatureTree, error) {
```

Comment:
Same comment as above.  Technically, this should probably be `releaseFeatureInfo`, but then again, release feels redundant.